### PR TITLE
removed unused option

### PR DESCRIPTION
**Pull request by JensWehner**

### CODE REVIEW

**Comment by JensWehner** [comment]
@votca-bot format

File: src/libtools/application.cc [hunk 1]
```diff
@@ -66,7 +66,6 @@ int Application::Exec(int argc, char **argv) {
     AddProgramOptions()("verbose1", "  be very loud and noisy");
     AddProgramOptions()("verbose2,v", "  be extremly loud and noisy");
     AddProgramOptions("Hidden")("man", "  output man-formatted manual pages");
```
Comment (junghans) [inline src/libtools/application.cc:68]: Is the `man` option used at all?

Comment (JensWehner) [reply src/libtools/application.cc:68]: I thought you use that for something.

Comment (junghans) [reply src/libtools/application.cc:68]: No, we use that new `votca_help2doc` script everywhere now.

Comment (JensWehner) [reply src/libtools/application.cc:68]: I thought that converts the man files to sth else.

Comment (junghans) [reply src/libtools/application.cc:68]: It only uses the output of `--help`.

Comment (JensWehner) [reply src/libtools/application.cc:68]: wow then we can remove quite a bit of stuff. :)

**Comment by JensWehner** [comment]
@junghans can you review again

**Review by junghans — APPROVED** [review]

**Pull request merged**